### ISSUE TITLE
Task: 3d mapping

### ISSUE THREAD

**Issue by Stalfur** [Stalfur/Askja#29]
If a place has buildings mapped, without 3d tags (roof:shape for example) but has Mapillary images, point it out as a possible 3D mapping project.

**Comment by waldyrious** [comment]
For the record, I'd make use of this feature :)

**Comment by Stalfur** [comment]
Yeah I did my first 3D mapping yesterday, just some roof stuff since I didn't have meter numbers ready. Looking at Notre Dame was scary, I have no idea what is happening there!
